nestkernel/event.h

```diff
@@ -746,63 +746,32 @@ class SecondaryEvent : public Event
   virtual fwit& operator>>( fwit& pos ) = 0;
 };
 
-/**
- * Iterator class for the GapJEvent data array.
- * Generating a copy of the array at several times is too time consuming,
- * therefore we use iterators pointing to the "old" memory location of the array
- */
-class CoeffArrayIterator
+template < typename T >
+size_t size_uint_t( T )
```

Comment:
@janhahne Could you add a Doxygen comment explaining the purpose of this function?
Also,  I am not entirely happy with the function name. "size" evokes associations to `sizeof()`, which returns the size of an object in bytes. This method, though, returns the number of uints required to hold an object of type `T`. Maybe `number_of_uints_covered()` would be better? Then the places where the function is used would become
```
pos += number_of_units_covered( d );
```